### PR TITLE
2021 Changes

### LINKED ISSUE #32: Having the code projects in this repo is confusing to new users
WPILib's VSCode extension expects code projects to top-level folders in the workspace and will fail to build them if it is not. Cloning this repo from vscode will result in the code projects not being top-level folders and as a result they won't build. The docs do currently say to "open" the relevant code projects but don't explain how to make it a top-level folder. None of this would be necessary if the code projects were just in other repos.

### PR DESCRIPTION
**Pull request by CoolSpy3**
Contains all updates and clarifications to training for 2021.

### CODE REVIEW

**Comment by CoolSpy3** [comment]
We might want to consider adding a SparkMax to the practice board and making it part of `docs/section-3/commands-subsystems.md`. However, I'm waiting on confirmation from SenAct before I make this change in the docs.

File: docs/section-4/auton.md [hunk 1]
```diff
@@ -17,4 +17,4 @@ More requirements for autonomous:
 - it can only use the encoder values to tell the command whether or not you have traveled far enough
 - it should not overshoot by a wide margin, but as long as it goes past 2 feet it's acceptable
 
-Good luck! Again, once you're done, check in with a programming lead or mentor then run your code on a robot!
\ No newline at end of file
+Good luck! Again, once you're done, simulate it and then check in with a programming lead or mentor then run your code on a robot!
```
Comment (brettle) [inline docs/section-4/auton.md:20]: Fix minor grammar error.

File: docs/section-5/delegation.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## Github Issues
+Bugs or features which have to be worked on/implemented should have an issue open on Github. An issue should contain a descriptive title which gives a good overview of the issue and a description which contains a thoughough description of the problem as well as any other relevant information.
```
Comment (brettle) [inline docs/section-5/delegation.md:2]: thoughough -> thorough

**Review by brettle — APPROVED** [review]
Nice work! Just a couple minor grammar/spelling errors to fix. Otherwise, good to merge.

File: docs/section-2/code-structure.md [hunk 3]
```diff
@@ -45,13 +45,15 @@ This is where we create and store the objects for the subsystems, then set up al
 This file contains all of the numbers that don't change and are relevant to the whole robot. This includes all ports and joystick buttons, which correlate to motor controllers, solenoids, and commands in the code.
 
 ## Subsystems
-Subsystems are anoter name for robot mechanisms, such as a drivetrain, an elevator, an intake, or a climber. The subystem classes contain the Objects for the motor controllers and solenoids and have methods for controlling the subsystem, which are used by commands.
+Subsystems are where much of the logic of the code is stored. In each subsystem, there are methods that perform certain actions, such as running a motor. These methods are then called in Commands as part of a larger process. For example, a method in the Climber subsystem could be used to retrieve the height of a lift, which can then be called in one of the Climber commands. The subsystem classes contain the Objects for the motor controllers and solenoids. Constants for the specific subsystem are also stored in the subsystem class. For example, a constant such as the speed of a drivetrain motor should be stored in the Drivetrain subsystem, as it’s only relevant to the Drivetrain.
```
Comment (XavierLem) [inline docs/section-2/code-structure.md:48]:
```suggestion
Subsystems are where much of the logic of the code is stored. In each subsystem, there are methods that perform certain actions, such as running a motor. These methods are then called in Commands as part of a larger process. For example, a method in the Climber subsystem could be used to retrieve the height of a lift, which can then be called in one of the Climber commands. The subsystem classes contain the Objects for the motor controllers and solenoids. Constants for a specific subsystem are also stored in the subsystem's class. For example, a constant such as the speed of a drivetrain motor should be stored in the Drivetrain subsystem, as it’s only relevant to the Drivetrain.
```
Make sentence more clear

**Review by XavierLem — APPROVED** [review]
I made one grammar suggestion for clarity, but everything else looks good to me.

**Pull request merged**